# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/transforms/rotated-clip-large.png
# Commit: b04340bad99d
# Full Hash: b04340bad99d70638efda2e7d528e89df77e47c4
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-05-22 09:43:16
# Description:
#   Bug 1617524 - Fix crash in get_relative_transform edge case. r=jnicol
#   
#   Previously, WR would attempt to detect at the start of frame
#   building if the spatial node of any picture cache contained
#   a non-axis-aligned transform, and disable picture caching in
# ==============================================================================

diff -r fdfb16cb82a6 -r b04340bad99d gfx/wr/wrench/reftests/transforms/rotated-clip-large.png
Binary file gfx/wr/wrench/reftests/transforms/rotated-clip-large.png has changed